resources/frontend_client/index_template.html

```diff
@@ -8,20 +8,15 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
     <link rel="apple-touch-icon" sizes="180x180" href="/app/assets/img/apple-touch-icon.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="/app/assets/img/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="/app/assets/img/favicon-16x16.png">
+    <link rel="icon" href="{{{favicon}}}" />
     <link rel="manifest" href="/app/assets/img/site.webmanifest">
-    <link rel="mask-icon" href="/app/assets/img/safari-pinned-tab.svg" color="#62a8e6">
```

Comment:
From [the article](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs):
### Safari Pinned Icon
Safari used to have a separate requirement to display an icon in pinned tabs. However, since Safari 12, we can use a regular favicon for pinned tabs. Even apple.com doesn’t use the `mask-icon` anymore.